Cingulara/openrmf-api-template
# release

- updating the version
- fixing the upgrade note and not caching it
- updated README
- #34 adding caching to API calls
- adding newer checklist templates
- updating tests for CI/CD with GH Actions
- updated STIG templates
- #32 adding a better health check
- adding Jaeger Tracing framework
- #27 STIG checklist updates and check latest
- #27 STIG checklist updates and check latest
- updating to 0.12
- #28 deleting templates with auditing
- #22 return the template record just added
- saving the uploaded description
- adding the version of the checklist
- #26 read in templates w/o SeverityOverride and CCI
- Fixing the title to be standard
- Final update of checklists XCCDF manual files
- First round of XCCDF files to make checklists
- putting the latest version tag in here
- add the filename to query from benchmarks
- Fixing uploaded template titles
- Fixing formatting if bad dups not used in viewer
- updating the loader to parse the rules correctly
- changing to manual XCCDF to generate templates
- added more newer checklists
- Adding newer Templates to work with on SCAP
- Adding Jaeger Tracing Information
- Add Template filename and version
- exporting Prometheus metrics
- updated copyright notice
- adding logging throughout the API calls
- fixing comments and spacing
- updating API descriptions
- updating the description of the API calls
- updated copyright notice
- Fixing version number
- added a count for user and system templates
- Update issue templates
- Updating to version 0.10
- Update to 0.9.1 on title standards fix
- standardizing titles on import
- Fixing Dockerfile as the project copies templates
- fixing the readme info
- updated code comments and version 0.9
- Added more SYSTEM Templates
- Updating the latest SYSTEM Templates for OpenRMF
- Updating the STIG checklists for system templates
- Cleaning up the checklist raw info
- #28 creating SYSTEM templates for use in SCAP
- fixing ENV to be all caps
- updating NATS IP
- fixing code for RBAC and API to API comms
- updating info to version 0.8
- RBAC updates and tests
- moving model tests to right folder
- reorg for unit tests
- #1 Fixing templates showing score
- update to openRMF
- make dockerhub faster
- updating for verison 0.6
- adding the checklist output to cycle through VulnIDs in the UI
- displaying raw checklist
- adjusting variables to match the rest of the APIs
- adding a health controller
- Adding the update on templates and the InternalId as the main piece
- #6 add remaining STIG types that are not behind PKI
- updating Makefile and fixing download template
- updating for getting the template record and recording logs
- adding updated time on creation
- adding a makefile for automation
- Updating STIG types
- Updating license to GPL3
- updating connection info
- updating readme and project info
- Dockerfile #2 made
- Adding Reading / Listing capability
